src/napari_imagej/widgets/menu.py

```diff
@@ -255,6 +266,45 @@ def disable_popup(self):
         )
 
 
+class REPLButton(IJMenuButton):
+    def __init__(self, viewer: Viewer):
+        super().__init__(viewer)
+        self.viewer = viewer
+
+        self.setEnabled(False)
+
+        icon = QColoredSVGIcon(resource_path("repl"))
+        self.setIcon(icon.colored(theme=viewer.theme))
+
+        self.clicked.connect(self._toggle_repl)
+        self._widget = None
+
+    def _add_repl_to_dock(self):
+        from scyjava import jimport
+
+        ByteArrayOutputStream = jimport("java.io.ByteArrayOutputStream")
+        ScriptREPL = jimport("org.scijava.script.ScriptREPL")
```

Comment:
Can we put these imports in [`java.py`](https://github.com/imagej/napari-imagej/blob/75cedb62a87a8d8ec20a1669133c517bf48c0b6e/src/napari_imagej/java.py#L196), along with all of the other imports? It's kinda nice to use `jimport` in one place

Reply:
Also outdated.